src/UI/Http/Rest/Controller/Event/GetEventsController.php

```diff
@@ -7,7 +7,7 @@
 use App\Application\Query\Collection;
 use App\Application\Query\Event\GetEvents\GetEventsQuery;
 use App\UI\Http\Rest\Controller\QueryController;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\Routing\Annotation\Route;
```

Comment:
Whats the reason behind this?

Reply:
https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/CHANGELOG.md#52

The Route from FrameworkExtraBundle is deprecated in the new version.

Reply:
+1 for a separate PR for this